tests/test_utils.py

```diff
@@ -874,6 +874,7 @@ def plot_fluxnet_comparison(driver, science_test_data_dir,
 
     # loop over Ameriflux sites
     obs_dir = os.path.join(science_test_data_dir,
+                           'science',
```

Comment:
same here.  Please roll back any unrelated changes that came about via merge conflicts.